src/tools/stereoCalib/src/stereoCalibThread.cpp

```diff
@@ -130,6 +134,232 @@ void stereoCalibThread::run(){
         monoCalibRun();
     }
 }
+
+void stereoCalibThread::calibFromImages()
+{
+	// Avoid to start the common calibration
+	// TODO: use a semaphore or mutex to handle this
+	startCalibration=0;
+	
+	std::vector<std::string> leftImages;
+	std::vector<std::string> rightImages;
+	auto dir = opendir (cfiFolder.c_str());
+	yInfo() << cfiFolder;
+	dirent *ent;
+
+	// TODO: use yaro::os libraries to get the files names
+	if (dir != NULL) {
```

Comment:
Please use `nullptr` instead of `NULL`
https://stackoverflow.com/questions/20509734/null-vs-nullptr-why-was-it-replaced